api-docs/v2/api-operations/other-operations.rst

```diff
@@ -0,0 +1,8 @@
+.. _other-operations: 
+
+Other operations
+~~~~~~~~~~~~~~~~~~~
+
+Use these operations to manage other functions, like rate limits.
```

Comment:
Yeah, but our editor frowns on sections without introductory comments.  Ideally, we'll add more miscellaneous operations in time and then it'll make more sense!